src/evrythng/tests/test_entities.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+import os
+
+from evrythng.entities import actions
+from evrythng.entities import products
+
+API_KEY = os.environ['API_KEY']
+
+
+class TestProductMethods(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        """Product Tests Setup"""
+
+        cls.product_id = products.create_product(
+            name='test_creating_prod_GooeeIOT', api_key=API_KEY).json()['id']
+
+    @classmethod
+    def tearDownClass(cls):
+        """Product Tests Destroying"""
+
+        products.delete_product(
+            product_id=str(cls.product_id), api_key=API_KEY)
+
+    def test_creation(self):
+        """creating test product and
```

Comment:
I think the first sentence of a docstring should fit on one line with respect to max line width.  I usually use a summary for the first line and if need be I add more below it on a new line.